config/forms/2021_2022.json

```diff
@@ -2423,9 +2430,10 @@
                   "step": 1,
                   "width": 5,
                   "prefix": "£",
-                  "suffix": "every 2 weeks",
+                  "suffix": " every 2 weeks",
```

Comment:
Sorry little detail, but why do we need a space for suffixes?

Reply:
In the check answers page we construct the answer from prefix and suffix. I could join the prefix, answer and suffix with an empty space, but then there would be a space between `£` and amount. I could also join prefix and answer without space and join the suffix with space, but that then doesn't account for non currency questions that might have prefixes/suffixes. We don't have such questions yet, but in case we do in the future this was the easiest way. I can change it if you convince me to tho!